sql/core/src/main/scala/org/apache/spark/sql/execution/stat/FrequentItems.scala

```diff
@@ -90,7 +90,7 @@ private[sql] object FrequentItems extends Logging {
       (name, originalSchema.fields(index).dataType)
     }
 
-    val freqItems = df.select(cols.map(Column(_)) : _*).rdd.aggregate(countMaps)(
+    val freqItems = df.select(cols.map(Column(_)) : _*).internalRowRdd.aggregate(countMaps)(
```

Comment:
When we calculate `singlePassFreqItems`, we don't need to convert catalyst types to scala types before calculation. `DataFrame.rdd` is `RDD[Row]` and we need `RDD[InternalRow]` here.